views/pages/ConfigPage.jsx

```diff
@@ -1,20 +1,217 @@
-import React from 'react'
-import { useOutletContext } from 'react-router-dom'
+import React, { useState, useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+import { useOutletContext, Link } from 'react-router-dom'
+
+import AttachFile from '@material-ui/icons/AttachFile'
+import { Fab } from '@material-ui/core'
+import ContentAdd from '@material-ui/icons/Add'
+import Tooltip from '@material-ui/core/Tooltip'
 import ConfigurationsList from '../components/ConfigurationsList'
+import ShareForm from '../forms/ShareForm'
+import { UsersModel } from '../models'
+import api from '../api'
+import { routes } from '../routes/routes'
+import useGrid from '../hooks/useGrid'
```

Comment:
Not sure if you've been using a convention here but could be helpful to alphabetize the imports.

Reply:
[fe296b3](https://github.com/AMP-SCZ/dpdash/pull/334/commits/fe296b313788b3c528d89388989b6c3165760d61)